src/main/kotlin/no/nav/helsearbeidsgiver/oppgave/Oppgave.kt

```diff
@@ -0,0 +1,98 @@
+package no.nav.helsearbeidsgiver.oppgave
+
+import kotlinx.serialization.KSerializer
+import kotlinx.serialization.Serializable
+import kotlinx.serialization.descriptors.PrimitiveKind
+import kotlinx.serialization.descriptors.PrimitiveSerialDescriptor
+import kotlinx.serialization.descriptors.SerialDescriptor
+import kotlinx.serialization.encoding.Decoder
+import kotlinx.serialization.encoding.Encoder
+import java.time.LocalDate
+import java.time.format.DateTimeFormatter
+
+@Serializable
+data class Oppgave(
+    val id: Int? = null,
+    val versjon: Int? = null,
+    val tildeltEnhetsnr: String? = null,
+    val opprettetAvEnhetsnr: String? = null,
+    val aktoerId: String? = null,
+    val journalpostId: String? = null,
+    val behandlesAvApplikasjon: String? = null,
+    val saksreferanse: String? = null,
+    val tilordnetRessurs: String? = null,
+    val beskrivelse: String? = null,
+    val tema: String? = null,
+    val oppgavetype: String,
+    val behandlingstype: String? = null,
+    val behandlingstema: String? = null,
+    @Serializable(LocalDateSerializer::class)
+    val aktivDato: LocalDate,
+    @Serializable(LocalDateSerializer::class)
+    val fristFerdigstillelse: LocalDate? = null,
+    val prioritet: String,
+    val status: String? = null,
+    val mappeId: Int? = null,
+    val orgnr: String? = null,
+)
+
+@Serializable
+data class OpprettOppgaveRequest(
+    val tildeltEnhetsnr: String? = null,
+    val aktoerId: String? = null,
+    val journalpostId: String? = null,
+    val behandlesAvApplikasjon: String? = null,
+    val saksreferanse: String? = null,
+    val beskrivelse: String? = null,
+    val tema: String? = null,
+    val oppgavetype: String,
+    val behandlingstype: String? = null,
+    val behandlingstema: String? = null,
+    @Serializable(LocalDateSerializer::class)
+    val aktivDato: LocalDate,
+    @Serializable(LocalDateSerializer::class)
+    val fristFerdigstillelse: LocalDate? = null,
+    val prioritet: Prioritet,
+)
+
+@Serializable
+data class HentOppgaverRequest(
+    val oppgavetype: String?,
+    val tema: String?,
+    val behandlingstype: String?,
+    val behandlingstema: String?,
+    val statuskategori: Statuskategori?,
+    val tildeltEnhetsnr: String?,
+    val tilordnetRessurs: String?,
+    val journalpostId: String?,
+    val saksreferanse: String?,
+    val orgnr: String?,
+    val limit: Int?,
+    val offset: Int?,
+)
+
+@Serializable
+data class OpprettOppgaveResponse(
+    val id: Int,
+)
+
+@Serializable
+data class OppgaveListeResponse(
+    val antallTreffTotalt: Int,
+    val oppgaver: List<Oppgave>,
+)
+
+object LocalDateSerializer : KSerializer<LocalDate> {
```

Comment:
Jeg tenker at det er bedre å ta i bruk https://github.com/navikt/helsearbeidsgiver-utils
for dette. evt legge til det der hvis det mangler feks.

Reply:
eksempel på bruk :
https://github.com/navikt/helsearbeidsgiver-inntektsmelding/blob/39be7430530bf6dd82238390390956babdc5526a/felles/src/main/kotlin/no/nav/helsearbeidsgiver/felles/domene/Arbeidsforhold.kt